README.md

```diff
@@ -45,7 +45,11 @@ Session session = new SessionImpl("http://localhost:8080", new BasicAuthenticati
 Push.with(session).register(this, SENDER_ID);
 ```
 
-The `SENDER_ID` is the identifier of your GCM project, if you follow the official [Android's documentation](http://developer.android.com/google/gcm/client.html) it should be easy to obtain one.
+If you want to use Liferay 7.x you should manually specify the version with a call like this:
+
+```java
+push.withPortalVersion(70)
+```
```

Comment:
I want to change it with the builder, I should send a small PR or modify this one